server/src/main/java/com/paypal/selion/grid/servlets/SauceServlet.java

```diff
@@ -57,10 +77,47 @@ public SauceServlet() {
         this(null);
     }
 
-    /**
+    private String formatForHtmlTemplate(String message) {
+        return String.format("<p align='center'><b>%s</b></p>", message);
+    }
+
+    @Override
+    protected Registry getRegistry() {
+        // ensure the Registry returned reflects the hub state.
+        final Registry localRegistry = super.getRegistry();
+        final Registry hubRegistry = localRegistry.getHub().getRegistry();
+        // yes, we only care if they are the same object reference.
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Use equals() to compare object references.](https://www.codacy.com/app/SeLion/paypal-SeLion/file/2360780041/issues/source?bid=2928425&fileBranchId=3219779#l90)